packages/web/components/Player/Filter/AdjascentTimezonePlayers.tsx

```diff
@@ -0,0 +1,102 @@
+import { Flex, LoadingState, Text, TimezoneOptions, VStack } from '@metafam/ds';
+import { PlayerList } from 'components/Player/PlayerList';
+import { GetPlayersQueryVariables } from 'graphql/autogen/types';
+import { usePlayerFilter } from 'lib/hooks/players';
+import { useOnScreen } from 'lib/hooks/useOnScreen';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+
+type Props = {
+  queryVariables: GetPlayersQueryVariables;
+};
+
+const getAdjacentTimezoneQueryVariables = (
+  defaultQueryVariables: GetPlayersQueryVariables,
+): GetPlayersQueryVariables => {
+  const timezoneValue = defaultQueryVariables.timezones
+    ? defaultQueryVariables.timezones[0]
+    : undefined;
+  const timezone = TimezoneOptions.find((t) => t.value === timezoneValue);
+  const adjascentTimezones = timezone
+    ? TimezoneOptions.filter(
+        (t) =>
+          Math.abs(t.offset - timezone.offset) <= 4 &&
+          t.value !== timezoneValue,
+      ).map((t) => t.value)
+    : [];
+  return {
+    ...defaultQueryVariables,
+    offset: 0,
+    timezones: adjascentTimezones,
+  };
+};
+
+export const AdjascentTimezonePlayers: React.FC<Props> = ({
+  queryVariables,
+}) => {
+  const [variables, setVariables] = useState<GetPlayersQueryVariables>(
+    getAdjacentTimezoneQueryVariables(queryVariables),
+  );
+
+  useEffect(() => {
+    setVariables(getAdjacentTimezoneQueryVariables(queryVariables));
+  }, [queryVariables]);
+
+  const moreRef = useRef(null);
+  const onScreen = useOnScreen(moreRef);
+
+  const {
+    players,
+    totalCount,
+    fetching,
+    fetchingMore,
+    error,
+    nextPage,
+    moreAvailable,
+  } = usePlayerFilter(variables);
```

Comment:
I did some research on this and the way to include players w/o a profile in the query results is apparently to compose the where statement at this level and then pass that to the filter function rather than a list of variables.

I've been meaning to test it out, but haven't had the time yet.

I think requiring players to fill out their profile is always going to leave us with holes in the ranks.

Reply:
Sounds good. Perhaps we can get to this in a future PR